.travis.yml

```diff
@@ -1,11 +1,20 @@
 language: java
 
-install: mvn install -DskipTests=true -Dgpg.skip=true
+before_install:
+  - nvm install $NODE_VERSION
+  - curl -s "https://get.sdkman.io" | bash
+  - source "$HOME/.sdkman/bin/sdkman-init.sh"
+  - sdk install java 17.0.5-amzn
+  - sdk use java 17.0.5-amzn
+
+install:
+  - mvn install -DskipTests=true -Dgpg.skip=true
+  - npm install
 
 dist: trusty
 
-jdk:
-  - oraclejdk8
+#jdk:
+#  - oraclejdk8
```

Comment:
Why don't we have to tell travis to use oraclejdk17?

Reply:
It doesn't exist